docs/source/markdown/podman-create.1.md

```diff
@@ -488,7 +488,7 @@ Tune a container's memory swappiness behavior. Accepts an integer between 0 and
 
 Attach a filesystem mount to the container
 
-Current supported mount TYPES are `bind`, `volume`, and `tmpfs`.
+Current supported mount TYPES are `bind`, `volume`, and `tmpfs`. <sup>[[1]](#Footnote1)</sup>
```

Comment:
Do we need the footnote here?

Reply:
https://github.com/containers/podman/pull/6903#discussion_r451564180